tests/services/fontgen_test.py

```
@@ -0,0 +1,275 @@
+_CEMOJI = _find_cemoji()
+
+
+def _cemoji_renderable():
+    """The colour parity tests compare against the C `fontconvert`; probe *that*
+    binary (not the Python renderer, which now decodes colour without a PNG-enabled
+    FreeType) so the parity class skips rather than fails when the C tool's FreeType
+    lacks PNG."""
+    if _ERR is not None or _CEMOJI is None or _FONTCONVERT is None:
+        return False
+    try:
+        subprocess.run([_FONTCONVERT, "-f", _CEMOJI, "-s20", "-g", "-r40", "-b32",
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)